deploys.go

```diff
@@ -74,6 +74,42 @@ type DeployVersionResult struct {
 	Name string `json:"name"`
 }
 
+// DeployVersionListResult stores a list of deployment versions
+type DeployVersionListResult struct {
+	Versions []*DeployVersionResult `json:"versions"`
+}
+
+func newRespErr(response *http.Response, msg string) error {
```

Comment:
Wonderful. Could you move this to utils.go? I have been meaning to write a little something to make error handling a bit more descriptive and this is great.